nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/InvokeHTTP.java

```diff
@@ -669,6 +694,14 @@ public void onTrigger(ProcessContext context, ProcessSession session) throws Pro
         final int maxAttributeSize = context.getProperty(PROP_PUT_ATTRIBUTE_MAX_LENGTH).asInteger();
         final ComponentLog logger = getLogger();
 
+        // log ETag cache metrics
+        final boolean eTagEnabled = context.getProperty(PROP_USE_ETAG).asBoolean();
+        if(eTagEnabled) {
```

Comment:
I think `&& logger.isDebugEnabled()` should be added to this line to prevent the extra work being done since this is only for debugging.